ulog/log_test.go

```diff
@@ -161,3 +162,18 @@ func TestSentryHookDoesNotMutatePrevious(t *testing.T) {
 	assert.NotNil(t, l2.sh.Fields())
 	assert.Equal(t, map[string]interface{}{"key": "value"}, l2.sh.Fields())
 }
+
+func TestStackTraceLogger(t *testing.T) {
+	t.Parallel()
+	testutils.WithInMemoryLogger(t, nil, func(zapLogger zap.Logger, buf *testutils.TestBuffer) {
+		log := Builder().SetLogger(zapLogger).Build()
+		err1 := errors.New("for sure")
+		err2 := errors.Wrap(err1, "it's a trap")
+		log.Error("error message", "error", err2)
+		trace := `{"level":"error","msg":"error message","stacktrace":{"log_test.go":"line:171","in_memory_log.go":"line:60","log_test.go":"line:178"`
```

Comment:
We can, my only concern is readability: the line is going to be really big. I've already trimmed it to have only file name not a full path.
A function name is a bit useless in case if we have line numbers. We can change provided information based on a feedback.